scripts/templates/helper.py

```diff
@@ -1148,6 +1148,9 @@ def get_loader_prologue(namespace, tags, obj, meta):
             fty_name = re.sub(r"(\w+)_handle_t", r"\1_factory", tname)
 
             if type_traits.is_pointer(item['type']):
+                if not param_traits.is_range(item):
```

Comment:
I think this check should happen alongside other similar spec validation functions in `__validate_params` from `parse_spec.py`.

Reply:
👍